doc/nrf/doc_styleguide.rst

```diff
@@ -305,27 +305,38 @@ Here is an example of a fully defined function:
 Enums
 =====
 
-The documentation block must precede the documented element.
-This is in accordance with the :ref:`Zephyr coding style <zephyr:coding_style>`.
+You can place the documentation block above the documented element or inline.
```

Comment:
Can this be misread as "documented element or documented inline"?

Also, I think the enum needs to be documented above itself. It's the elements inside it that can be documented inline.

Reply:
Good point. Trying to make this clearer.